src-input/duk_hobject.h

```diff
@@ -36,6 +36,11 @@
  * inspection code.
  */
 
+/* FIXME: ES spec: all construct-only functions have [[Call]] and [[Construct]]
```

Comment:
Hehe, yes. I don't keep work in progress branches in a well commented state btw. That's why they're work in progress :)